app/src/main/res/layout-land/admin_pin_activity.xml

```diff
@@ -10,89 +10,103 @@
+          android:layout_height="wrap_content"
+          android:layout_marginStart="72dp"
+          android:layout_marginTop="16dp"
+          android:layout_marginEnd="72dp"
+          android:text="@string/admin_pin_pin_description"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="16sp"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@+id/admin_pin_main_text" />
 
-      <Button
-        android:id="@+id/submit_button"
-        style="@style/StateButtonActive"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="20dp"
-        android:layout_marginEnd="130dp"
-        android:text="@string/admin_pin_submit"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/input_confirm_pin" />
-    </androidx.constraintlayout.widget.ConstraintLayout>
-  </ScrollView>
+        <org.oppia.app.profile.ProfileInputView
+          android:id="@+id/input_pin"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="140dp"
+          android:layout_marginTop="16dp"
+          android:layout_marginEnd="140dp"
+          app:inputLength="5"
+          app:isPasswordInput="true"
+          app:label="@string/admin_pin_new_pin"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@+id/admin_pin_warning_text"
+          profile:error="@{viewModel.pinErrorMsg}" />
+
+        <org.oppia.app.profile.ProfileInputView
+          android:id="@+id/input_confirm_pin"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="140dp"
+          android:layout_marginEnd="140dp"
+          app:inputLength="5"
+          app:isPasswordInput="true"
+          app:label="@string/admin_pin_new_confirm"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintHorizontal_bias="0.0"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@+id/input_pin"
+          profile:error="@{viewModel.confirmPinErrorMsg}" />
+
+        <Button
+          android:id="@+id/submit_button"
+          style="@style/StateButtonActive"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="48dp"
+          android:layout_marginEnd="28dp"
+          android:background="@{viewModel.isButtonActive() ? @drawable/state_button_primary_background : @drawable/start_button_transparent_background}"
+          android:clickable="@{viewModel.isButtonActive()}"
+          android:minHeight="48dp"
+          android:text="@string/admin_pin_submit"
```

Comment:
String says save. Can we change its id or is it ok to have it as submit only.

Reply:
Updated to `R.string.save` so that we can use this for general purpose in future.